### PR TITLE
October pre release feadback fixes

### PR DESCRIPTION
**Pull request by mechack2022**
Request to merge Octerber pre release feadback fixes

### CODE REVIEW

File: src/main/java/org/lamisplus/modules/hiv/config/AsyncConfiguration.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/config/AsyncConfiguration.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/config/DatabaseInitializer.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/config/DatabaseInitializer.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/controller/HivEnrollmentController.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/controller/HivEnrollmentController.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/controller/OvcLinkageController.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/controller/OvcLinkageController.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/domain/dto/PatientFlagConfigsDto.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/domain/dto/PatientFlagConfigsDto.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/domain/dto/TransferPatientInfo.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/domain/dto/TransferPatientInfo.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/repositories/HIVEacRepository.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/repositories/HIVEacRepository.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/repositories/HivEnrollmentRepository.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/repositories/HivEnrollmentRepository.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/repositories/PatientFlagRepository.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/repositories/PatientFlagRepository.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/service/HIVEacSessionService.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/service/HIVEacSessionService.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/service/HivPatientService.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/service/HivPatientService.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/service/LinkageService.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/service/LinkageService.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/service/PatientFlagService.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/service/PatientFlagService.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/service/StatusManagementService.java
Comment (AJ-DataFI) [inline src/main/java/org/lamisplus/modules/hiv/service/StatusManagementService.java:1]: remove commented code

File: src/main/java/org/lamisplus/modules/hiv/config/DatabaseInitializer.java
Comment (mechack2022) [reply src/main/java/org/lamisplus/modules/hiv/config/DatabaseInitializer.java:1]: done

File: src/main/java/org/lamisplus/modules/hiv/service/StatusManagementService.java
Comment (mechack2022) [reply src/main/java/org/lamisplus/modules/hiv/service/StatusManagementService.java:1]: done

**Pull request closed without merging**